lib/teiserver_web/live/admin/matchmaking/index.ex

```diff
@@ -0,0 +1,90 @@
+defmodule TeiserverWeb.Admin.MatchmakingLive.Index do
+  use TeiserverWeb, :live_view
+  alias Teiserver.Matchmaking
+
+  @impl true
+  def mount(_params, _session, socket) do
+    case allow?(socket.assigns[:current_user], "Admin") do
+      true ->
+        socket =
+          socket
+          |> assign(:site_menu_active, "matchmaking")
+          |> assign(:view_colour, Teiserver.Battle.MatchLib.colours())
+          |> assign(:queues, [])
+          |> add_breadcrumb(name: "Admin", url: "/teiserver/admin")
+          |> add_breadcrumb(name: "Matchmaking", url: "/admin/matchmaking")
+
+        :timer.send_interval(5_000, :refresh_queues)
+
+        {:ok, socket}
+
+      false ->
+        {:ok,
+         socket
+         |> redirect(to: ~p"/")}
+    end
+  end
+
+  @impl true
+  def handle_params(_params, _url, socket) do
+    {:noreply, get_queues(socket)}
+  end
+
+  @impl true
+  def handle_info(:refresh_queues, socket) do
+    :timer.send_interval(5_000, :refresh_queues)
```

Comment:
`:send_interval` sends a message every `n` ms. Calling that every time mean every 5s, you are doubling the amount of message sent to the process.
Either use one `send_interval` once in init, or `send_later` in init and the handler.